src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -70,12 +70,15 @@ export function PolarisTestProvider({
   frame,
 }: PolarisTestProviderProps) {
   const Wrapper = strict ? StrictMode : Fragment;
-  const intl = new I18n(i18n || {});
-  const scrollLockManager = new ScrollLockManager();
+  const intl = useMemo(() => new I18n(i18n || {}), [i18n]);
```

Comment:
Eslint warns with [react/jsx-no-constructed-context-values](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md) when you sick something in a context that shall cause the context to be updated on every single rerender regardless of if the content changes or not. With the fix being to memoize objects you pass into contexts.

In this particular case it's probably not a big deal as we probably aren't causing rerenders on PolarisTestProvider because, but it's an easy fix to write code in the way to keep the linter happy rather than sprinkling `eslint-disable-next-line`s around